.github/workflows/test.yml

```diff
@@ -6,12 +6,16 @@ on:
   pull_request:
     types: [opened, synchronize, reopened]
 jobs:
+  read-node-version:
+    name: "Read NodeJS version"
+    runs-on: ubuntu-22.04
+    steps:
+      - name: "Read .nvmrc"
+        run: "echo NVMRC=`cat .nvmrc` >> $GITHUB_ENV"
```

Comment:
You need to checkout the repo before reading the version in order for the file to be available for the runner.

Reply:
https://github.com/canonical/maas-react-components/blob/main/.github/workflows/CI.yml